seqr/views/apis/dashboard_api_tests.py

```diff
@@ -120,3 +126,57 @@ def test_export_projects_table(self):
         response = self.client.get('{}?file_format=csv'.format(url))
         self.assertEqual(response.status_code, 400)
         self.assertDictEqual(response.json(), {'error': 'Invalid file_format: csv'})
+
+
+if anvil_enabled():
+    @mock.patch('seqr.views.utils.terra_api_utils.GOOGLE_SERVICE_ACCOUNT_INFO', GOOGLE_SERVICE_ACCOUNT_INFO)
+    class DashboardPageAnvilTest(DashboardPageTest):
+        fixtures = ['users', 'social_auth_data', '1kg_project']
+
+        @responses.activate
+        def test_dashboard_page_data(self):
+            responses.add(responses.POST, GOOGLE_ACCESS_TOKEN_URL, status=200, body=GOOGLE_TOKEN_RESULT)
+            responses.add(responses.POST, GOOGLE_API_TOKEN_URL, status=200, body=GOOGLE_TOKEN_RESULT)
+            responses.add(responses.GET, WORKSPACE_WITH_FIELDS_URL, status=200, body=WORKSPACE_RSP_NO_VALID_PROJECT)
+            responses.add(responses.GET, WORKSPACE2_ACL_URL, status=200, body=WORKSPACE2_ACL_RSP)
```

Comment:
These are the wrappers that requested for the existing tests to run with `TERRA_API` enabled.